lib/utils/createElement.js

```diff
@@ -9,8 +10,8 @@ export function toggleButton(
 	onClick?: (enabling: boolean) => void = () => {},
 	fieldID: string,
 	enabled?: boolean = false,
-	onText?: string | HTMLElement = 'on',
-	offText?: string | HTMLElement = 'off',
+	onText?: string | HTMLElement = i18n('toggleOn'),
```

Comment:
I feel like the `i18n` should be applied inside the function to `.onText` and `.offText`, instead of only to the default value. Then we can put simply i18n keys into the `onText` and `offText` properties in the code and rely on them being magically translated as late as possible.